application/core/auth_core/src/jvmMain/kotlin/io/writeopia/auth/core/repository/EncryptedTokenStorage.kt

```
@@ -0,0 +1,240 @@
+     */
+    private fun getMachineSpecificKey(): String {
+        val components = mutableListOf<String>()
+
+        // MAC address
+        try {
+            val localHost = InetAddress.getLocalHost()
+            val networkInterface = NetworkInterface.getByInetAddress(localHost)
+            networkInterface?.hardwareAddress?.let { mac ->
+                components.add(mac.joinToString("-") { "%02X".format(it) })
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)